src/Command/Playercount.js

```diff
@@ -0,0 +1,115 @@
+const Command = require('../Command');
+
+const messages = {
+    'errorChecking': 'Hmm.. I encountered some issues looking up the players.  Is Shotbow.net offline?',
+    'errorBadKey': 'Hmm.. I couldn\'t find the game you were talking about.  Try again?',
+    'result': 'There are currently %1 players connected to %2.',
+    'help': 'You can use `!playercount` to show the players connected to the network or some of the games.\nYou can use any of the following names: %1'
+};
+
+const keyAlias = {
+    'hcf': 'hcfactions',
+    'anni': 'annihilation',
+    'civ-war': 'civwar',
+    'civcraft': 'civwar',
+    'minetheftauto': 'mta',
+    'gc': 'ghostcraft',
+    'ghost': 'ghostcraft',
+    'mine theft auto': 'mta',
+    'death by void': 'dbv',
+};
+
+// Map of key to name - also acts as a whitelist
+const gameNames = {
+    'all': 'Shotbow',
+    'annihilation': 'Annihilation',
+    'minez': 'MineZ',
+    'hcfactions': 'HCFactions',
+    'dbv': 'Death by Void',
+    'smash': 'SMASH',
+    'slaughter': 'Slaughter',
+    'civwar': 'Civ-War',
+    'gg': 'GG',
+    'mta': 'Mine Theft Auto',
+    'ghostcraft': 'GhostCraft',
+    'lobby': 'Lobby',
+};
+
+const cacheTTL = 10 * 1000; // 10 seconds in milliseconds
+
+module.exports = Command.extend({
+    commandName: 'playercount',
+    https: null,
+    cache: null,
+    cacheKey: 'serverlist',
+    dependencies: {
+        'discordClient': 'discordClient',
+        'commandPrefix': 'commandPrefix',
+        'https': 'https',
+        'Cache': 'cache'
+    },
+    processMessage: function (message, tokens) {
+        this.fetchServerlist(serverList => {
+            if (serverList === false) {
+                message.channel.send(messages.errorChecking);
+                return;
+            }
+            tokens.shift();
+            let key = this.getGameKey(tokens.join(' '));
```

Comment:
key should be trimmed in case they put whitespace after the parameter: `tokens.join(' ').trim()`